snap/snapcraft.yaml

```diff
@@ -407,3 +407,14 @@ parts:
       ## and the NDDSHOME environment variable has not been set. Support for Connext will not be available.
       #--ros-jazzy-rmw-connext-cpp
       #--ros-jazzy-rmw-gurumdds-cpp
+  humble-desktop-libamdhip-exec-stack:
```

Comment:
This part is named `humble-...` but the pr targets `jazzy`. Mind to rename that ? :)

Reply:
I noticed, but going by [this fix](https://github.com/ubuntu-robotics/ros-content-sharing-snaps/pull/16/files) where the if condition checks for jazzy desktop and names the part humble. If there's no reason for that, I'll rename

Reply:
Well, it should be fixed over there too :sweat_smile: . Please use `jazzy` here.